app/src/main/java/ch/epfl/sdp/drone3d/MappingMissionSelectionActivity.kt

```diff
@@ -53,15 +64,45 @@ class MappingMissionSelectionActivity : AppCompatActivity() {
 
     }
 
+
+    private fun setListenerMappingMissions(){
+        val ownerId = authService.getCurrentSession()!!.user.uid
+
+        val liveSharedMappingMissions = mappingMissionDao.getSharedMappingMissions()
+        liveSharedMappingMissions.observe(this, Observer {
+            mappingMissionSharedList = it.toMutableList()
+            if(currentType == StorageType.SHARED){
+                updateList()
+            }
+        })
+
+        val livePrivateMappingMissions = mappingMissionDao.getPrivateMappingMissions(ownerId)
+        livePrivateMappingMissions.observe(this, Observer {
+            mappingMissionPrivateList = it.toMutableList()
+            if(currentType == StorageType.PRIVATE){
```

Comment:
The format is not perfect, it lacks a space between the if statement and the bracket

Reply:
Fixed